server/controllers/follow.js

```diff
@@ -164,7 +161,7 @@ export default class Follow {
         }
       }
     } catch (err) {
-      response(res).sendData(500, err);
+      response(res).serverError({ message: 'Couldnt unfollow user' });
```

Comment:
Could you restructure this message appropriately, the absence of a quote in couldn't is wrong

Reply:
Thanks, I'll change it